interfaces/dbus/backend.go

```diff
@@ -46,6 +50,29 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 	return "dbus"
 }
 
+// setupHostDBusSessionConf will ensure that we have a dbus configuration
+// that points to /var/lib/snapd/dbus/services. This is needed for systems
+// that re-exec snapd and do not have this configuration as part of the
+// packaged snapd.
+func setupHostDBusSessionConf() error {
+	dbusSessionConf := filepath.Join(dirs.SnapSessionBusPolicyDir, "snapd.conf")
+	if osutil.FileExists(dbusSessionConf) {
```

Comment:
Could we use osutil.EnsureDirState for this? This will allow us to just have a simple call here that ensures the _correct_ content is in that file.